examples/unix_socket/source/app.d

```diff
@@ -0,0 +1,13 @@
+import vibe.inet.url;
+import vibe.http.client;
+import vibe.stream.operations;
+import std.stdio;
+
+void main()
+{
+	URL url = URL("unix:///var/run/docker.sock") ~ Path("containers/json");
```

Comment:
This should be "http+unix:///var/run/docker.sock...", so that it's still possible to select the proper application protocol. Is there any precedence for such a URL? From what I've found, and so that the URL has a unique string representation (`URL(url.toString()) == url`), the best way to represent it is `"http+unix://%2Fvar%2Frun%2Fdocker.sock/containers/json"` and then using `urlDecode` on the `url.host` value before putting it in the UNIX address struct.